File: src/main/java/com/my/admin/app/commerce/model/repository/CashStatusRepository.java
package com.my.admin.app.commerce.model.repository;

import com.black.food.manager.model.entity.CashStatus;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface CashStatusRepository extends JpaRepository<CashStatus, Long> {
}

File: src/main/kotlin/com/bk/reactive/app/commerce/admin/my/repository/CashStatusRepository.kt
package com.bk.reactive.app.commerce.admin.my.repository

import com.bk.reactive.app.commerce.admin.my.model.document.CashStatus
import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository

@Repository
interface CashStatusRepository : ReactiveMongoRepository<CashStatus, String> {
}